runtime/doc/options.txt

```diff
@@ -4547,6 +4540,19 @@ A jump table for the options with a short description can be found at |Q_op|.
 <
 	UI-dependent. Works best with RGB colors. 'termguicolors'
 
+						*'pumheight'* *'ph'*
+'pumheight' 'ph'	number	(default 0)
+			global
+	Determines the maximum number of items to show in the popup menu for
+	Insert mode completion.  When zero as much space as available is used.
+	|ins-completion-menu|.
+
+						*'pumwidth'* *'pw'*
+'pumwidth' 'pw'		number	(default 15)
+			global
+	Determines the minium width to use for the popup menu for Insert mode
+	completion.  |ins-completion-menu|.
```

Comment:
would be great to also mention your comment in the doc here:

> complete() `startcol`  + `&pumwidth` must be less than the screen width to display the pum at the `startcol`.

something like:

> If the cursor column  + 'pumwidth' exceed screen width, the popup menu is nudged to fit on the screen.

Reply:
done in https://github.com/neovim/neovim/pull/11604